packages/bpk-component-split-input/src/BpkInputField.module.scss

```diff
@@ -0,0 +1,34 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* stylelint-disable */
+@import '~bpk-mixins';
+
+.BpkInputField {
+  display: flex;
+  max-width: 6.688rem;
```

Comment:
where does `6.688rem` come from?

Reply:
ahh, I see. It's ok if there's no bpk token to match the exact value, we should still be using the tokens and multiply/add in order to get the value we want. This might be hard to achieve with this value though 🤔 does it break if it's rounded up/down?